src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -183,14 +184,16 @@ public void ActiveSolutionBoundTracker_Changes()
 
             using (var testSubject = new ActiveSolutionBoundTracker(this.host, this.activeSolutionTracker, loggerMock.Object))
             {
-                var solutionBindingChangedEventCount = 0;
-                testSubject.SolutionBindingChanged += (obj, args) => { solutionBindingChangedEventCount++; };
+                var eventCounter = new EventCounter(testSubject);
```

Comment:
Added a new helper class to make registering for the events and checking they are raised/not raised simpler.